src/WPF/ArcGISRuntime.WPF.Viewer/Samples/GraphicsOverlay/AddGraphicsRenderer/AddGraphicsRenderer.xaml.cs

```diff
@@ -42,9 +42,29 @@ private void Initialize()
                 MakeLineGraphicsOverlay(),
                 MakeSquareGraphicsOverlay(),
                 MakeCurvedGraphicsOverlay(),
+                MakeEllipseGraphicsOverlay()
             });
         }
 
+        private GraphicsOverlay MakeEllipseGraphicsOverlay()
+        {
+            // Create a simple fill symbol with outline.
+            SimpleLineSymbol curvedLineSymbol = new SimpleLineSymbol(SimpleLineSymbolStyle.Solid, Color.Purple, 1);
+            SimpleFillSymbol curvedFillSymbol = new SimpleFillSymbol(SimpleFillSymbolStyle.Solid, Color.Purple, curvedLineSymbol);
+
+            // Create a graphics overlay for the polygons with curve segments.
+            GraphicsOverlay ellipseGraphicsOverlay = new GraphicsOverlay();
+
+            // Create and assign a simple renderer to the graphics overlay.
+            ellipseGraphicsOverlay.Renderer = new SimpleRenderer(curvedFillSymbol);
+
+            //Makes ellipse graphic
```

Comment:
For this kind of sample, I think the preference is to add a comment that explains every line or couple of lines, e.g.

```cs
// Create the polygon geometry, then create a graphic using that polygon.
Polygon ellipse = MakeEllipseGeometry();
Graphic ellipseGraphic = new Graphic(ellipse);

// Add the graphic to the graphics overlay.
ellipseGraphicsOverlay.Graphics.Add(ellipseGraphic);

return ellipseGraphicsOverlay;
```